.github/workflows/integration_tests.yml

```diff
@@ -184,7 +184,7 @@ jobs:
           create_function_result=$(ngc-cli/ngc cloud-function function create \
             --container-image nvcr.io/${{ env.NGC_QUANTUM_ORG }}/${{ env.NGC_QUANTUM_TEAM }}/cuda-quantum:nightly \
             --container-environment-variable NUM_GPUS:1 \
-            --container-environment-variable NVQC_REST_PAYLOAD_VERSION:1 \
+            --container-environment-variable NVQC_REST_PAYLOAD_VERSION:2 \
```

Comment:
I forget - was this change going to get pulled into a separate PR? Or was it just that we were going to hold off on this full PR until closer to the release.

Reply:
We will close this PR and migrate the changes to a new branch based on `main` with deployment to test the integration. The changes will be merged closer to release.